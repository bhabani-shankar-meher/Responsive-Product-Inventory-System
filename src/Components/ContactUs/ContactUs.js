import React from "react";
import Box from "@mui/material/Box";
import Grid from "@mui/material/Grid";
import Divider from "@mui/material/Divider";
import Typography from "@mui/material/Typography";
const ContactUs = () => {
    return (
        <Box sx={{ width: "80%", marginLeft: "10%" }} mt={5} mb={3}>
            <Grid
                container
                rowSpacing={3}
                columnSpacing={{ xs: 1, sm: 2, md: 3 }}
            >
                <Grid item xs={12} md={12}>
                    <Typography textAlign="center" variant="h4">
                        Contact Us
                    </Typography>
                    <Divider sx={{ width: "100%", marginTop: "20px" }} />
                </Grid>
                <Grid item xs={12} md={12}>
                    <Typography textAlign="center" variant="h6">
                        If you have or issue or query please write mail to us <a href="mailto:bhabani.meher@wipro.com">bhabani.meher@wipro.com</a>
                    </Typography>
                </Grid>
                <Grid item xs={12} md={12}>
                    <Typography textAlign="center" variant="h6">
                        Or you can reach out to us by dialing <a href="#">7606044523</a>
                    </Typography>
                </Grid>
            </Grid>
        </Box>
    );
};
export default ContactUs;