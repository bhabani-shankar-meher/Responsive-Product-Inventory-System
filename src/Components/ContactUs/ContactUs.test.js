import ContactUs from "./ContactUs";
import { render, screen } from "@testing-library/react";
import { create } from "react-test-renderer";

describe("CONTACTUS PAGE SNAPSHOT", () => {
    test("ContactUs Page Snapshot Executed Successfully", () => {
        const contactUsSnapshot = create(<ContactUs />);
        expect(contactUsSnapshot).toMatchSnapshot();
    });
});
describe("CONTACTUS PAGE TEST", () => {
    test("ContactUs Page Heading Test", () => {
        render(<ContactUs />);
        expect(
            screen.getByRole("heading", {
                name: "Contact Us",
            })
        ).toBeInTheDocument();
    });
    test("ContactUs Page Test_1", () => {
        render(<ContactUs />);
        expect(
            screen.getByRole("heading", {
                name: "If you have or issue or query please write mail to us bhabani.meher@wipro.com",
            })
        ).toBeInTheDocument();
    });
    test("ContactUs Page Test_2", () => {
        render(<ContactUs />);
        expect(
            screen.getByRole("heading", {
                name: "Or you can reach out to us by dialing 7606044523",
            })
        ).toBeInTheDocument();
    });
    test("ContactUs Page Test_3", () => {
        render(<ContactUs />);
        expect(screen.getByText('bhabani.meher@wipro.com').closest('a')).toHaveAttribute('href', 'mailto:bhabani.meher@wipro.com')
    });
    test("ContactUs Page Test_4", () => {
        render(<ContactUs />);
        expect(screen.getByText('7606044523').closest('a')).toHaveAttribute('href', '#')
    });
});