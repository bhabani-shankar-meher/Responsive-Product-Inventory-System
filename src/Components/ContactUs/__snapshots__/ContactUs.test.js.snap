// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CONTACTUS PAGE SNAPSHOT ContactUs Page Snapshot Executed Successfully 1`] = `
<div
  className="MuiBox-root css-1mo2yih"
>
  <div
    className="MuiGrid-root MuiGrid-container css-qwfiov-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 css-1idn90j-MuiGrid-root"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4 css-149xwtl-MuiTypography-root"
      >
        Contact Us
      </h4>
      <hr
        className="MuiDivider-root MuiDivider-fullWidth css-1dh14ht-MuiDivider-root"
      />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 css-1idn90j-MuiGrid-root"
    >
      <h6
        className="MuiTypography-root MuiTypography-h6 css-1kgxqm0-MuiTypography-root"
      >
        If you have or issue or query please write mail to us 
        <a
          href="mailto:bhabani.meher@wipro.com"
        >
          bhabani.meher@wipro.com
        </a>
      </h6>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 css-1idn90j-MuiGrid-root"
    >
      <h6
        className="MuiTypography-root MuiTypography-h6 css-1kgxqm0-MuiTypography-root"
      >
        Or you can reach out to us by dialing 
        <a
          href="#"
        >
          7606044523
        </a>
      </h6>
    </div>
  </div>
</div>
`;
